packages/serialize/src/index.js

```diff
@@ -374,7 +374,7 @@ export const serializeStyles = function(
       args[i],
       styles.charCodeAt(styles.length - 1) === 46
     )
-    if (stringMode) {
+    if (stringMode && i < strings.length) {
```

Comment:
This requires a new test to be added. I don't think we need to assert `console.error` being called or not, but it seems that this has also caused `"undefined"` being appended to the styles, so it would be good to test that it no longer happens.

If you need any help with setting up a repository for the local development or if you are unsure how exactly such a test should be added then please reach out to me on our Slack so I could provide the assistance for you.